### PR TITLE
klarna-2.3.5-updates

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request

This pull request (PR) updates the Klarna topics to reflect the following changes in the 2.3.5 release:
- Updated Changes In this Release
- Updated Klarna configuration settings and instructions. The names of configuration sections and field labels were updated for consistency.
- Updated configuration screenshots.

## Affected documentation pages
[Klarna](https://docs.magento.com/m2/ee/user_guide/payment/klarna.html) (intro)
[Setting Up Klarna](https://docs.magento.com/m2/ee/user_guide/payment/klarna-setup.html)
[Klarna](https://docs.magento.com/m2/ee/user_guide/configuration/sales/klarna.html) (configuration)
[Change Log](https://docs.magento.com/m2/ee/user_guide/magento/change-log.html)

## Affected Magento editions
- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information
Staging: #430

### CODE REVIEW

File: src/payment/klarna.md [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
-Klarna lets your customers choose how they want to pay, and supports the purchase with buyer protection and excellent customer service.
+Klarna lets your customers choose how they want to pay, and supports the purchase with buyer protection and excellent customer service. Your customers can buy now and pay later, while you receive payment in full at the time of purchase.
 
 ## Changes In the Latest Release
 
-In the Magento configuration, the Klarna Payments section has a new "Data sharing on load" field that can be set to share customer data after the transaction is authorized, or when the Klarna payment method is selected during checkout.
+### New Features
+
+-  Added support for Australia and New Zealand markets
+-  Improved configuration: Changed scope of API Comfiguration from `Store View` to `Website;` streamlined many field labels.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/payment/klarna.md:12]: misspelling:  `Comfiguration`

File: src/payment/klarna-setup.md [hunk 2]
```diff
@@ -62,36 +62,39 @@ _Klarna Merchant Portal_
    ![Klarna]({% link images/images/config-sales-payment-methods-recommended-solutions-klarna.png %}){: .zoom}
    [_Klarna_]({% link configuration/sales/klarna.md %})
 
-1. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **Klarna API** section. Then, do the following:
+1. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **API Configuration** section. Then, do the following:
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/payment/klarna-setup.md:65]: no -- expander labels should remain bold

File: src/configuration/sales/klarna.md [hunk 3]
```diff
@@ -4,18 +4,18 @@ title: Klarna
 
 Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Sales]({% link configuration/sales.md %}) > [Payment Methods]({% link configuration/sales/payment-methods.md %}) > Klarna
```
[the remark this answers is not in the document]
Comment (ccondra) [reply src/configuration/sales/klarna.md:5]: No, this is how the Configuration Reference paths are formatted.

File: src/payment/klarna.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ccondra) [reply src/payment/klarna.md]: Fixed.

**Pull request merged**